helpers/arguments.py

```diff
@@ -774,12 +774,13 @@ def parse_args(input_args=None):
     parser.add_argument(
         "--resolution_type",
         type=str,
-        default="area",
-        choices=["pixel", "area"],
+        default="pixel_area",
+        choices=["pixel", "area", "pixel_area"],
         help=(
             "Resizing images maintains aspect ratio. This defines the resizing strategy."
-            " If 'pixel', the images will be resized to the resolution by pixel edge."
+            " If 'pixel', the images will be resized to the resolution by the shortest pixel edge, if the target size does not match the current size."
             " If 'area', the images will be resized so the pixel area is this many megapixels."
```

Comment:
I believe this should be transparent about the 'silent' overrides. Something like?
```suggestion
            " If 'area', the images will be resized so the pixel area is this many megapixels. Common rounded values such as `0.5` and `1.0` will be implicitly adjusted to their squared size equivalents."
```